src/models/ISigUpParams.ts

```diff
@@ -0,0 +1,11 @@
+import { InputName } from '../utils/consts';
```

Comment:
❗️
Правильнее хранить константы в файле `constants` и не в `utils` папке, а в `src`
```suggestion
import { InputName } from '../utils/constants';
```

Reply:
Исправлено !!